### PR TITLE
 [#116785861] Create /developer/myapp/new page where users can register their app

### PR DESCRIPTION
**Pull request by ademola-raimi**
#### What does this PR do?

Create a page where user can register their application
#### Description of Task to be completed?

/developers/myapp/new route presents a page with a simple form to fill in all the required details like Application name, Homepage Url. Application Description and a Register Application button.
#### How should this be manually tested?

From /developer/myapp page, click on Create A New App button. This takes you to /developer/myapp/new which contains the simple form. See the below to view the screenshot

 #### Any background context you want to provide?

None
#### What are the relevant pivotal tracker stories?
#116785861
#### Screenshots (if appropriate)

screen shot 2016-04-05 at 11 16 55 am
#### Questions:

[@unicodeveloper](https://github.com/unicodeveloper)

### CODE REVIEW

File: database/migrations/2016_04_04_193247_create_newApps_table.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+class CreateNewAppsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('newApps', function (Blueprint $table) {
```
Comment (andela-oadebayo) [inline database/migrations/2016_04_04_193247_create_newApps_table.php:15]: **Code Quality - Critical**
@andela-araimi The table name here is not explanatory enough, change it to a better name and also change the column names to be make them simpler.

File: app/NewApp.php [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Suyabay;
+
+use Illuminate\Database\Eloquent\Model;
+
+class NewApp extends Model
```
Comment (andela-oadebayo) [inline app/NewApp.php:7]: **Code Quality - Critical**
@andela-araimi This Model name is very confusing and will not pass the right message to a future Dev. on this project. Please endeavour to change to a much better and self explanatory name.

**Comment by ademola-raimi** [comment]
@andela-oadebayo: I think you can merge this PR now, I have implemented the helper function 'old' as discussed. The other issues are not meant for this PR, I will implement them in my next PR.

File: app/Http/routes.php [hunk 3]
```diff
@@ -10,6 +10,9 @@
 Route::get('/developer', 'ApiPagesController@index');
 Route::get('/developer/myapp', 'ApiPagesController@myApp');
 Route::get('/developer/myapp/new', 'ApiPagesController@createNewApp');
+Route::post('/developer/myapp/new/', function() {
+    echo "App created successfully";
+});
```
Comment (unicodeveloper) [inline app/Http/routes.php:15]: **Code Functionality - Critcal**

Is there any reason we have it echoing a sentence on a post request?

**Comment by ademola-raimi** [comment]
@unicodeveloper: I used the echo to test the 'old function'. In the next PR, of course, the echo will definitely be removed.

**Comment by unicodeveloper** [comment]
Why wait till the next PR, Please remove it now! @andela-araimi

**Comment by unicodeveloper** [comment]
Please rebase to fix the merge conflicts @andela-araimi

**Comment by andela-oadebayo** [comment]
**Code Quality - Critical**
@andela-araimi You said you made some changes which we could not verify with a commit, please create a commit to fix this issue.

**Pull request merged**